src/Authoring/WinRT.SourceGenerator/Generator.cs

```diff
@@ -59,12 +59,13 @@ private void GenerateSources()
             string winmds = context.GetCsWinRTDependentMetadata();

```

Comment:
Any reason to make `csWinRTExeTFM` a parameter rather than a variable here like the others.